src/main/java/cs361/battleships/models/Board.java

```diff
@@ -112,13 +111,13 @@ private Result attack(Square s) {
 		}
 		Ship hitShip; // = shipsAtLocation.get(0);
 		if (shipsAtLocation.get(0).getKind().equals("MINESWEEPER")){
-			hitShip = (Minesweeper)shipsAtLocation.get(0);
+			hitShip = shipsAtLocation.get(0);
 		}
 		else if (shipsAtLocation.get(0).getKind().equals("DESTROYER")){
-			hitShip = (Destroyer)shipsAtLocation.get(0);
+			hitShip = shipsAtLocation.get(0);
 		}
 		else {
-			hitShip = (Battleship)shipsAtLocation.get(0);
+			hitShip = shipsAtLocation.get(0);
```

Comment:
Do you think this if statement is necessary since they all do the same thing?

Reply:
Nope you're right, I removed it